ios/web/BUILD.gn

```diff
@@ -14,3 +14,13 @@ test("ios_brave_web_unittests") {
     "//brave/ios/web/webui:ios_web_webui_unittests",
   ]
 }
+
+test("ios_brave_web_inttests") {
```

Comment:
Not sure if we want to include integration tests by default in the `brave_all_unit_tests` list